htdocs/expedition/class/expedition.class.php

```diff
@@ -238,7 +245,7 @@ function create($user)
 		$sql.= ", ".(!empty($this->model_pdf)?"'".$this->db->escape($this->model_pdf)."'":"null");
 		$sql.= ")";
 
-		dol_syslog(get_class($this)."::create", LOG_DEBUG);
+		dol_syslog(get_class($this)."::create sql=".$sql, LOG_DEBUG);
```

Comment:
If it is logged by driver, what is the sense keeping ?

dol_syslog(get_class($this)."::create", LOG_DEBUG);

Correct it again...

Le 02/04/2015 01:02, Marcos García a écrit :
> dol_syslog(get_class($this)."::create", LOG_DEBUG);